src/editor/app.js

```diff
@@ -390,6 +390,26 @@ UiDriver.registerEventHandler("C_CMD_SET_THEME", function(msg, data, prevReturn)
     editor.setOption("theme", data.name);
 });
 
+UiDriver.registerEventHandler("C_CMD_SET_FONT", function(msg, data, prevReturn) {
+
+    var fontSize = (data.size != "" && data.size>0) ? ("font-size:" + data.size + "px;") : "";
+
+    var fontFamily = data.family != "" ? ("font-family:'" + data.family + "';") : "";
+
+    var styleTag = document.getElementById('userFont');
+
+    if( styleTag  ){ 
+        styleTag.innerHTML = ".editor > .CodeMirror { " + fontFamily + fontSize + " }";
+        styleTag.appendTo('head');
```

Comment:
Well I guess it's the same effect, lol.  I just worry about using 
innerHTML overwriting something important in other use cases.  It may 
not apply here though.